server/controller/entries.js

```diff
@@ -64,6 +64,23 @@ class EntriesController {
     }
   }
 
+  static async getEntries(req, res) {
+    const userId = req.decoded.userId;
+
+    try {
+      await models.Entries.findAll()
```

Comment:
if you are using await.. you dont need to chain a .then again.. Pls review this!